examples/asr/transcribe_speech.py

```diff
@@ -226,7 +237,10 @@ def main(cfg: TranscriptionConfig) -> TranscriptionConfig:
                 decoding_cfg.preserve_alignments = cfg.compute_timestamps
             if 'compute_langs' in decoding_cfg:
                 decoding_cfg.compute_langs = cfg.compute_langs
-            asr_model.change_decoding_strategy(decoding_cfg, decoder_type=cfg.decoder_type)
+            if isinstance(asr_model, EncDecHybridRNNTCTCModel):
```

Comment:
You can check if the model has cur_decoder instead of checking the model's type explicitly as we may add more models like this.